pkg/distributor/distributor.go

```diff
@@ -52,17 +52,19 @@ import (
 )
 
 const (
-	maxIngestionRateFlag        = "distributor.instance-limits.max-ingestion-rate"
-	maxInflightPushRequestsFlag = "distributor.instance-limits.max-inflight-push-requests"
+	maxIngestionRateFlag             = "distributor.instance-limits.max-ingestion-rate"
+	maxInflightPushRequestsFlag      = "distributor.instance-limits.max-inflight-push-requests"
+	maxInflightPushRequestsBytesFlag = "distributor.instance-limits.max-inflight-push-requests-total-bytes"
```

Comment:
nit: Most other places don't include "total" but the flag does. Should this be changed to be consistent?